jupyter-mime.el

```diff
@@ -313,6 +313,17 @@ aligns on the current line."
         (setf (image-property image :ascent) 50)
         (force-window-update)))))
 
+(defun jupyter--insert-temp-file-browse-url (data)
```

Comment:
Since you use this function in `jupyter-R.el` and is general enough that it could be used in other situations, it should really be made into a "public" function by removing the double dash and adding a doc string.

Reply:
I've renamed to `jupyter-browse-url-in-temp-file` and added a docstring. Not sure it belongs in `jupyter-mime.el` still, but couldn't find a better place to put it, so I've left it there.